tests/test_cef_formatter.js

```
@@ -191,8 +194,64 @@ var suite = vows.describe("Formatter")
       }
     }
   }
+})
+
+.addBatch({
+    "Extension formatting": {
+        topic: function() {
+          var config = {
+            vendor: "Initech",
+            product: "Red Stapler",
+            version: "2"
+          };
+          return new Formatter(config);
+        },
+
+        "with invalid values": {
+            topic: function(formatter) {
+                var params = {
+                  name: "Low on staples",
+                  signature: "17",
+                  severity: 6,
+                  extensions: {
+                    rt: "20130320",
+                  }
+                };
+                return formatter.format(params);
+            },
+
+            "throws an error": function(_, err, result) {
+                assert(err);
+                assert(!result);
```

See https://github.com/jedp/node-cef/pull/6#r3639989 , I'm leaning towards being intolerant of errors in the code and to just fail fast. We can pass off errors to a raven client and log them to sentry.  I think we have a better chance of getting debug information out of our apps with a full stacktrace.